CONTRIBUTING_MODELS.md

```diff
@@ -72,168 +119,174 @@ contribution! 🙂
-such as number of layers, number of attention heads; preprocessor-specific
-arguments such as whether we want to lowercase the input text; checkpoint and
-vocabulary file URLs, etc. In the PR description, you can add
-Google Drive/personal GCP bucket links to the checkpoint and the vocabulary
-files. These files will then be uploaded to GCP by us!
+- Provide a script that converts weights from their original format (e.g., 
+PyTorch .bin, TensorFlow SavedModel) to the Keras H5 format expected by KerasHub.
+- This script should be reusable and clearly documented.
+- It's crucial for verifying weight conversion accuracy and for future updates.
+End-to-End Validation Colab
+- This is the most important validation step.
 
-After wrapping up the preset configuration file, you need to
-add the `from_preset` function to all three classes, i.e., `DistilBertBackbone`,
-and `DistilBertTokenizer`. Here is an
-[example](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_backbone.py#L187-L189).
+#### End-to-End Colab
 
-The testing for presets is divided into two: "large" and "extra large".
-For "large" tests, we pick the smallest preset (in terms of number of parameters)
-and verify whether the output is correct. For "extra large tests", we loop over
-all the presets and just check whether the backbone and the tokenizer can
-be called without any error.
+- Load task model using `from_preset()`
+- Run task (e.g., classification, generation)
+- Compare output with original model
 
-Additionally, a checkpoint conversion script should be added. This script
-demonstrates that the outputs of our backbone model and outputs of the source
-model match. This should be done for all presets.
+#### Numerics Test
 
-### Step 5: PR #4 and Beyond: Add XXTasks and XXPreprocessors
+- Add a test that loads a preset and compares outputs on fixed inputs
 
-Once you are finished with Steps 1-4, you can add "task" models and
-preprocessors.
-
-### Task model
+---
 
-Task models are essentially models which have "task heads" on top of the backbone
-models. For instance, for the text classification task, you can have a
-feedforward layer on top of a backbone model like DistilBERT. Task models are
-very essential since pretrained models are used extensively for downstream tasks
-like text classification, token classification, text summarization, neural
-machine translation, etc.
+### Step 6: PR #5 and Beyond – Advanced Features (Optional)
 
-#### Preprocessor
+Extend utility:
 
-The preprocessor class is responsible for making the inputs suitable for
-consumption by the model - it packs multiple inputs together, i.e., given
-multiple input texts, it will add appropriate special tokens, pad the inputs
-and return the dictionary in the form expected by the model.
+- New Task Models (e.g., TokenClassifier, ImageSegmentation)
+- Parameter-Efficient Fine-Tuning (LoRA support)
+- Quantization (QLoRA support)
```

Comment:
what do we expect to be added for lora and qlora support? this seems kinda ill defined